posthog/temporal/tests/batch_exports/test_batch_exports.py

```diff
@@ -29,9 +29,9 @@ def assert_records_match_events(records, events):
     all_record = sorted(records, key=operator.itemgetter("event"))
 
     assert len(all_expected) == len(all_record)
-    assert len([record["uuid"] for record in all_record]) == len(
-        {record["uuid"] for record in all_record}
-    ), "duplicate records found"
+    assert len([record["uuid"] for record in all_record]) == len({record["uuid"] for record in all_record}), (
+        "duplicate records found"
+    )
```

Comment:
My computer keeps modifying these files that are not part of this PR, perhaps in a hook?